forloop_modules/queries/db_model_templates.py

```diff
@@ -156,7 +170,7 @@ class APINodeJob(BaseModel):
     created_at: UTCDatetime
     completed_at: Optional[UTCDatetime] = None
     message: Optional[str] = None
-    node_uid: str
+    node: APINode
```

Comment:
I dont like nesting / composing of objects like this too much, it should rather be sequentially queried in my opinion